pyodide_build/buildpkg.py

```diff
@@ -300,7 +301,7 @@ def make_parser(parser):
     return parser
 
 
-def main(args):
+def main(args: List(Any)):
```

Comment:
I'm not sure this is right. After a quick glance I wasn't able to find the return type of `ArgumentParser.parse_args`. It's also fine to leave it untyped.